app/experimenter/nimbus-ui/src/components/PageNew/index.test.tsx

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import PageNew from ".";
+
+describe("PageNew", () => {
+  let origConsoleLog: typeof global.console.log;
+
+  beforeEach(() => {
+    origConsoleLog = global.console.log;
+    global.console.log = jest.fn();
+  });
+
+  afterEach(() => {
+    global.console.log = origConsoleLog;
+  });
+
+  it("renders as expected", async () => {
+    render(<Subject />);
+    expect(screen.getByTestId("PageNew")).toBeInTheDocument();
+  });
+
+  it("handles experiment form submission", () => {
+    render(<Subject />);
+    fireEvent.click(screen.getByTestId("submit"));
+    expect(global.console.log).toHaveBeenCalledWith("CREATE TBD", {
```

Comment:
Wrote these tests for "TBD" console messages to get 100% coverage, but I'm expecting they'll be replaced to cover calls to actual APIs in the near future.